src/server/ProcessList.cpp

```diff
@@ -210,67 +186,48 @@ ConsoleProcessHandle* ConsoleProcessList::FindProcessByGroupId(_In_ ULONG ulProc
 // - E_OUTOFMEMORY in a low memory situation.
 [[nodiscard]] HRESULT ConsoleProcessList::GetTerminationRecordsByGroupId(const DWORD dwLimitingProcessId,
                                                                          const bool fCtrlClose,
-                                                                         _Outptr_result_buffer_all_(*pcRecords) ConsoleProcessTerminationRecord** prgRecords,
-                                                                         _Out_ size_t* const pcRecords) const
+                                                                         std::vector<ConsoleProcessTerminationRecord>& termRecords) const

```

Comment:
```suggestion
                                                                         _Out_ std::vector<ConsoleProcessTerminationRecord>& termRecords) const
```
nit: if you want to match the SAL annotations